pages/upload.vue

```diff
@@ -159,23 +152,18 @@ export default {
       isExceedMaxSize: false,
       isImageLoading: false,
       isShowUploadImageForm: false,
-      isUnsupportedFormat: false,
-      acceptInput: SUPPORTED_FILE_TYPES.reduce(
-        (acc, current) => `image/${acc}, ${current}`
-      ),
+      acceptInput: SUPPORTED_FILE_TYPES.map((e) => `image/${e}`).join(', '),
```

Comment:
I think the `SUPPORTED_FILE_TYPES` it self should already in the format of `image/png`, `image/jpeg` or `image/webp`.